src/components/molecules/FilterMenu/FilterMenuFooter/FilterMenuFooter.tsx

```diff
@@ -0,0 +1,17 @@
+import {StyledOkButton, StyledResetButton, StyledSpace} from './FilterMenuFooter.styled';
+
+type Props = {
```

Comment:
Rename Props to FilterMenuFooterProps, please

Reply:
done